web/pwa/src/app/page/storage/subpage-directory/subpage-directory.component.html

```diff
@@ -14,6 +14,16 @@
         <ion-icon name="ellipsis-vertical-outline"></ion-icon>
       </ion-button>
     </ion-buttons>
+	
+	<ion-buttons slot="secondary">
```

Comment:
My original intention for moving the "Help" to the menu on this view was to maximise the space available for the current folder in the toolbar. In particular, the back button on iOS is larger and comes with a text label. Do you think this is too awkward for the user, or can we live without it.

Reply:
Hmmm... I think the only problem is that when the ? is not visible in the new view, the user may think there is no help available. Up to you, I think it would be ok to keep it.